haas/cli.py

```diff
@@ -135,10 +156,19 @@ def init_db():
     server.init(init_db=True)
 
 @cmd
-def user_create(username, password):
-    """Create a user <username> with password <password>."""
-    url = object_url('user', username)
-    do_put(url, data={'password': password})
+def user_create(username, password, is_admin):
+    """Create a user <username> with password <password>.
+
+    <is_admin> may be either "admin" or "no-admin", and determines whether
+    the user has administrative priveledges.
+    """
+    url = object_url('/auth/basic/user', username)
+    if is_admin not in ('admin', 'no-admin'):
+        raise TypeError
```

Comment:
Done, though there's some weirdness around how that exception is used; see the commit messages.